distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/ImagingTest.java

```diff
@@ -155,11 +140,278 @@ public void testImageNitfChipCreation() throws Exception {
+     */
+    @Test
+    public void testNitfJpeg2000p012a() throws Exception {
+        ingestAndValidateCommonNitfJpeg2000Attributes("p0_12a").and()
+                .hasStringElement("resource-size", "1852")
+                .hasDateElement("ext.nitf.image.imageDateAndTime", "2001-12-13T11:15:18.000+00:00")
+                .hasStringElement("checksum", "fa66cc4e");
+    }
+
+    /**
+     * The feature or item tested in this file is: "5-3 Saturation test"
+     */
+    @Test
+    public void testNitfJpeg2000p014b() throws Exception {
+        ingestAndValidateCommonNitfJpeg2000Attributes("p0_14b").and()
+                .hasStringElement("resource-size", "2507")
+                .hasDateElement("ext.nitf.image.imageDateAndTime", "2001-12-17T17:36:26.000+00:00")
+                .hasStringElement("checksum", "9cf98387");
+    }
+
+    /**
+     * The feature or item tested in this file is: "Empty packet header bit"
+     */
+    @Test
+    public void testNitfJpeg2000p016a() throws Exception {
+        ingestAndValidateCommonNitfJpeg2000Attributes("p0_16a").and()
+                .hasStringElement("resource-size", "8974")
+                .hasDateElement("ext.nitf.image.imageDateAndTime", "2002-01-23T19:32:10.000+00:00")
+                .hasStringElement("checksum", "9e81c514");
+    }
+
+    /**
+     * The feature or item tested in this file is: "Reset context probabilities, vertically causal
+     * contexts, precinct sizes, PPT marker segment"
+     */
+    @Test
+    public void testNitfJpeg2000p102b() throws Exception {
+        ingestAndValidateCommonNitfJpeg2000Attributes("p1_02b").and()
+                .hasStringElement("resource-size", "263963")
+                .hasDateElement("ext.nitf.image.imageDateAndTime", "2001-12-26T15:02:10.000+00:00")
+                .hasStringElement("checksum", "74b85e69");
     }
 
-    private static void deleteMetacard(String id) {
+    /**
+     * The feature or item tested in this file is: "QCD marker segment in tile header
+     */
+    @Test
+    public void testNitfJpeg2000p104a() throws Exception {
+        ingestAndValidateCommonNitfJpeg2000Attributes("p1_04a").and()
+                .hasStringElement("resource-size", "103411")
+                .hasDateElement("ext.nitf.image.imageDateAndTime", "2001-11-02T06:30:50.000+00:00")
+                .hasStringElement("checksum", "1dc68130");
+    }
+
+    /**
+     * The feature or item tested in this file is: "Small tile size" (3 by 3)
+     */
+    @Test
+    public void testNitfJpeg2000p106b() throws Exception {
+        ingestAndValidateCommonNitfJpeg2000Attributes("p1_06b").and()
+                .hasStringElement("resource-size", "4229")
+                .hasDateElement("ext.nitf.image.imageDateAndTime", "2001-11-02T06:30:50.000+00:00")
+                .hasStringElement("checksum", "30c9ef90");
+    }
+
+    @After
+    public void tearDown() {
+        metacardIds.forEach(this::deleteMetacard);
```

Comment:
No need to change because this is nitpicky, but couldn't you use a String field for a single metacard id instead of a list since this method will run after each individual test?